scripts/ctype_translator.py

```diff
@@ -34,6 +34,7 @@
 # - LGLSXP: logical vectors
 # - INTSXP: integer vectors
 # - REALSXP: numeric vectors
+# - SINGLESXP: single precision numeric vectors
```

Comment:
I cannot find SINGLESXP on the list from the comment above https://cran.r-project.org/doc/manuals/r-release/R-ints.html

Reply:
Added a reference to relevant header file